crates/modelardb_server/src/storage/uncompressed_data_buffer.rs

```diff
@@ -31,13 +31,13 @@ use datafusion::arrow::compute;
 use datafusion::arrow::record_batch::RecordBatch;
 use datafusion::parquet::errors::ParquetError;
 use modelardb_common::metadata::model_table_metadata::ModelTableMetadata;
+use modelardb_common::metadata::univariate_id_to_column_index;
```

Comment:
Maybe only import `metadata` and use the function as `metadata::univariate_id_to_column_index()` for consistency.

Reply:
Fixed.